### PR TITLE
[FIX] issue with rewriting values on update

### PR DESCRIPTION
**Pull request by ArtyomLosev**

### CODE REVIEW

**Comment by ilmir-k** [comment]
@yelizariev check the code please
Is that right if we duplicate code from `ir_config_parameter.py` to `pre_install.yml` ?

**Comment by yelizariev** [comment]
Why?

**Comment by yelizariev** [comment]
default

**Comment by yelizariev** [comment]
@ArtyomLosev you need to update create_debranding_parameters instead and don't update value if one already exist

File: web_debranding/pre_install.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 -
   !python {model: ir.config_parameter}: |
     self.create_debranding_parameters()
+
```
Comment (yelizariev) [inline web_debranding/pre_install.yml:4]: revert this

**Pull request closed without merging**